pegasus/sites.v3/code.org/views/tenth_anniversary_tweets.html

```diff
@@ -0,0 +1,13 @@
+<div class="tenth-anniversary-tweets">
+  <blockquote class="twitter-tweet" data-dnt="true"><p lang="en" dir="ltr">Ten years ago TODAY, <a href="https://twitter.com/codeorg?ref_src=twsrc%5Etfw">@codeorg</a> was born! It’s been an awesome decade of inspiring future tech leaders and working to close the computer science opportunity gap! As education transforms in the age of AI, we hope to continue empowering students and teachers everywhere. 💻🎉 <a href="https://t.co/5IcreGlGXM">pic.twitter.com/5IcreGlGXM</a></p>&mdash; Code.org (@codeorg) <a href="https://twitter.com/codeorg/status/1629843842554789888?ref_src=twsrc%5Etfw">February 26, 2023</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+  <blockquote class="twitter-tweet" data-dnt="true"><p lang="en" dir="ltr">The <a href="https://twitter.com/codeorg?ref_src=twsrc%5Etfw">@codeorg</a> team is celebrating a decade of empowering students to explore computer science! Here&#39;s to many more years of transforming education! ❤️ <a href="https://t.co/MeeIOTjUaD">pic.twitter.com/MeeIOTjUaD</a></p>&mdash; Code.org (@codeorg) <a href="https://twitter.com/codeorg/status/1630257137732698112?ref_src=twsrc%5Etfw">February 27, 2023</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
Does this ref_src allow us to track clicks or something?

Reply:
That's a good question, I tried googling what the `ref_src` does, but didn't find anything relevant — I asked our social media folks and will report back.

Reply:
Heard back, the analytics platform we currently use doesn't have any specific info about tracking these embeds either.

Reply:
Got it. I'd be in favor of shortening it to use "https://twitter.com/codeorg" to (a) keep it simpler, and (b) avoid any unwanted data analytics happening on Twitter's end

Reply:
My fault –– I'm seeing this pattern here https://developer.twitter.com/en/docs/twitter-for-websites/embedded-tweets/overview which makes me think Twitter needs this for something

Reply:
This is the tool I use to get these embedded tweets: https://publish.twitter.com/# — I check the `Opt-out of tailoring Twitter` option which I think helps prevent tracking outside of whatever they need to use to make the tweet show up. 
<img width="613" alt="Screenshot 2023-06-09 at 8 29 18 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/e5a9fa6d-6f06-411d-bed5-53a824dbc93a">